check_addon.py

```diff
@@ -349,3 +351,48 @@ def _logWarning(error_counter, warning_string):
     comments_warning.append(warning_string)
     error_counter["warnings"] = error_counter["warnings"] + 1
     return error_counter
+
+
+
+
+def get_addons(xml_url):
+    """
+        addon.xml for the target Kodi version
+    """
+    content = requests.get(xml_url).content
+    tree = ET.fromstring(content).getroot()
+
+    return {
+        a.get("id"): a.get("version")
+        for a in tree.findall("addon")
+    }
+
+
+def get_users_dependencies(xml_file):
+    """
+        User's addon.xml from pull request
+    """
+
+    tree = ET.parse(xml_file).getroot()
+
+    return {
+        i.get("addon"): i.get("version")
+        for i in tree.findall("requires/import")
+    }
+
+def check_dependencies(error_counter, addon_xml):
+
+    branch_url = ROOT_URL.format(branch=os.environ.get("TRAVIS_BRANCH"))
```

Comment:
This will cause this to only work on CI, it should be possible to run locally to. Probably after setting some variables.

Reply:
should I make it usable via `.tests-config.json` ?

Reply:
Yes, do that. We might have to rethink that config file in the long run. But not now :)

Reply:
@Razzeee I have made it usable via .tests-config.json